the-exterminators/error-annihilator
# Sp 80 connect database

## Linked Issues

## PR Description

## Changes


## How to test

1. Fetch build and run
2. Try creating a new ticket
3. Make sure thet no error indicating that the db connection failed appears in the Terminal
4. Check if a notification appears confirming your ticket has been created and that the contents of the form are cleared
5. Check if your ticket was created in the db

### Expected behaviour:

## Checklist author


## Checklist reviewer/test-engineer
